opendps/func_settings.c

```diff
@@ -367,106 +343,112 @@ static set_param_status_t get_parameter(char *name, char *value, uint32_t value_
  * Get / Set callback functions
  */
 static int32_t get_v_adc_k() {
-    return v_adc_k_coef * 1000;
+    return v_adc_k_coef * 10000;
```

Comment:
Maybe this magic number should go into a define with an explanation.

Reply:
I can document this more. It's to offset the si_decimilli unit which is 10^-4.